tests/Fixtures/TestBundle/Entity/UuidIdentifierDummy.php

```diff
@@ -17,6 +17,8 @@
 /**
  * Custom identifier dummy.
  *
+ * @author Exploit.cz <insekticid@exploit.cz>
```

Comment:
@insekticid I have no problems with the `@author` tag but you made no changes on this class! 😄 
Add it to the `UpperCaseIdentifierDummy` class instead, it's more relevant! 😉